libexec/bootstrap-scripts/deffile-driver-docker.sh

```diff
@@ -52,7 +52,10 @@ fi
 SINGULARITY_CONTAINER="docker://$FROM"
 SINGULARITY_LABELFILE="$SINGULARITY_ROOTFS/.singularity.d/labels.json"
 
-SINGULARITY_CONTENTS=`mktemp /tmp/.singularity-layers.XXXXXXXX`
+if ! SINGULARITY_CONTENTS=`mktemp ${TMPIR:-/tmp}/.singularity-layers.XXXXXXXX`; then
```

Comment:
There are some typos here on the `TMPDIR` (`TMPIR`) that @bauerm97 just spotted with me. Can you correct those?